R/rlistings.R

```diff
@@ -143,6 +145,7 @@ as_listing <- function(df,
                        unique_rows = FALSE,
                        default_formatting = list(all = fmt_config()),
                        col_formatting = NULL,
+                       align_coltitles_as_cols = FALSE,
```

Comment:
Argument name is a bit wordy - maybe just `align_col_titles` or `align_colnames` or `align_labels`? I think the param description is explanatory enough. What do you think?

Reply:
it makes sense! Lets go with `align_colnames`. Thanks Emily ;)